afs/afs-ws/afs-ws-client-utils/src/main/java/com/powsybl/afs/ws/client/utils/RemoteServiceConfig.java

```diff
@@ -92,6 +111,42 @@ public RemoteServiceConfig setSecure(boolean secure) {
         return this;
     }
 
+    public boolean isAutoreconnect() {
```

Comment:
refactor at least to isAutoReconnect (auto and reconnect are two different words). Maybe you should had « Enabled »

Reply:
Ok, it is done, the new method is isAutoReconnectionEnabled